src/test/suite/backwardCompatibility.test.ts

```diff
@@ -1,16 +1,11 @@
 import * as assert from "assert";
 import * as vscode from "vscode";
-import * as sinon from "sinon";
 import { getCursorlessApi } from "../../util/getExtensionApi";
 import { openNewEditor } from "../openNewEditor";
+import { standardSuiteSetup } from "./standardSuiteSetup";
 
 suite("Backward compatibility", async function () {
-  this.timeout("100s");
-  this.retries(3);
-
-  teardown(() => {
-    sinon.restore();
-  });
+  standardSuiteSetup(this);
```

Comment:
Notice how for some tests we were retrying 3 times, other times 5.  Put this standard preamble into a utility function to encourage consistency